cluster-autoscaler/simulator/cluster_test.go

```diff
@@ -38,13 +38,13 @@ func TestUtilization(t *testing.T) {
 	node := BuildTestNode("node1", 2000, 2000000)
 	SetNodeReadyState(node, true, time.Time{})
 
-	utilInfo, err := CalculateUtilization(node, nodeInfo)
+	utilInfo, err := CalculateUtilization(node, nodeInfo, false)
 	assert.NoError(t, err)
 	assert.InEpsilon(t, 2.0/10, utilInfo.Utilization, 0.01)
 
 	node2 := BuildTestNode("node1", 2000, -1)
 
-	_, err = CalculateUtilization(node2, nodeInfo)
+	_, err = CalculateUtilization(node2, nodeInfo, false)
```

Comment:
Can you add a test case for ignoring DaemonSet pod?

Reply:
👍 Tests added for ignoring DaemonSet pods